api/src/main/java/com/okta/sdk/resource/ResourceException.java

```diff
@@ -35,9 +35,9 @@
     /**
      * Ensures the message used for the exception (i.e. exception.getMessage()) reports the {@code developerMessage}
      * returned by the Okta API Server.  The regular Okta response body {@code message} field is targeted
-     * at applicadtion end-users that could very likely be non-technical.  Since an exception should be helpful to
+     * at application end-users that could very likely be non-technical.  Since an exception should be helpful to
      * developers, it is better to show a more technical message.
-     * <p/>
+     * <p>
```

Comment:
Looks like we accidentally removed: `</p>`

Reply:
Intentional, it fails to lint this way (you cannot have self closing tags):
https://stackoverflow.com/questions/33787753/javadoc-closing-tags